src/tools/schema.js

```diff
@@ -115,17 +115,13 @@ const compileApp = appRaw => {
   });
 
   if (problemKeys.length) {
-    // TODO - DB: throw an error instead of logging
-    console.log(
-      '\nWARNING! The following key(s) conflict with those created by a resource:\n'
-    );
-    console.log(
+    const message = [
+      'The following key(s) conflict with those created by a resource:\n',
       problemKeys.map(k => `* ${k}`).join('\n'),
-      `\n\nEdit the standalone object${
-        problemKeys.length > 1 ? 's' : ''
-      } to resolve`,
-      '!! In the next major version, this will throw an error\n'
-    );
+      '\n\nEdit the standalone object(s) to resolve'
+    ].join('');
```

Comment:
> Edit the standalone object(s) to resolve

Minor, but the message seems a bit unclear to me. Maybe we can change it to "Rename the key(s) to resolve.".

Reply:
Sure. I just wanted to stress that you modify the standalone objects rather than the resources. Will fix tomorrow, merge eventually!